types/transform-pouch/transform-pouch-tests.ts

```diff
@@ -0,0 +1,27 @@
+import * as PouchDB from "pouchdb";
+import * as transform from "transform-pouch";
+PouchDB.plugin(transform);
+
+const db1 = new PouchDB<{example: number}>();
+db1.transform<{transformed: number}>({
+    incoming(doc) {
+        console.log(doc.example);
```

Comment:
```suggestion
```

Reply:
the console.log was used to just to have the doc.example in the code, to confirm the type exists.

Reply:
cool, but using `// $ExpectType T` is device for this and saves the CI cycles (console inroduced dependency)